extras/ghostty/modus_vivendi

```diff
@@ -0,0 +1,24 @@
+# Modus Themes for Ghostty
+# Auto generated with https://github.com/miikanissi/modus-themes.nvim/blob/master/lua/modus-themes/extras/ghostty.lua
+
```

Comment:
I don't think you need to include this to your PR commits.

Reply:
All the other files in the `extras` directory also have this comment, so I just followed that convention.

Reply:
I mean the file. Not the comments :)